### PR TITLE
Very Very basic layout

### LINKED ISSUE #2: Create a basic page layout
Create a layout of the frontend using **Typescript**, **React**, **TailwindCSS**, and **Next.js**.
Keep it a very simple page.

### PR DESCRIPTION
**Pull request by Lunchii**
Closes #2 
## Overview
white background with buttons (doesn't work) with dark blue bar at the top with text

### CODE REVIEW

File: app/page.tsx [hunk 1]
```diff
@@ -1,101 +1,20 @@
...
-            height={16}
-          />
-          Go to nextjs.org →
-        </a>
-      </footer>
-    </div>
+      <div>
+        <button className="bg-black float-left m-16 w-40 h-20">
+          Back
+        </button>
```
[the remark this answers is not in the document]
Comment (ZestyScrubz) [reply app/page.tsx:13]: Change font colour noob

**Review by Cheekane — DISMISSED** [review]
Basic stuff work. 👍

**Review by Cheekane — APPROVED** [review]
🦖

**Review by ZestyScrubz — APPROVED** [review]
It works, but it need work

**Review by jerpyk — APPROVED** [review]
seems alright

**Pull request merged**